frontend/.prettierrc

```diff
@@ -1,6 +1,7 @@
 {
   "printWidth": 120,
   "singleQuote": true,
+  "bracketSameLine": true,
```

Comment:
Why only on the "old" `frontend` package and not on all `packages/*/frontend/*/package.json`?

Reply:
I will update them.